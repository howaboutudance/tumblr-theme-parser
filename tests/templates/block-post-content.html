<html>
    <head>
    </head>
    <body>
        {block:Posts}
            <div class="post">
            <article class="{PostType}">
            {block:Text}
            {block:Title}
            <a href="{Permalink}">
                <h2>{Title}</h2> 
            </a>
            {/block:Title}
            {Body}
            {/block:Text}
            </div>
        {/block:Posts}
     </body>
</html>
