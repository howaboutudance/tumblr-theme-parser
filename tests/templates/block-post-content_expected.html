<html>
    <head>
    </head>
    <body>
        <div class="post">
            <article class="text">
            <a href="/post/1/">
                <h2>Title 1</h2> 
            </a>
            
            Body of the post 1.
            
            </div>
        <div class="post">
            <article class="text">
            <a href="/post/2/">
                <h2>Title 2</h2> 
            </a>
            
            Body of the post 2.
            
            </div>
        <div class="post">
            <article class="text">
            <a href="/post/3/">
                <h2>Title 3</h2> 
            </a>
            
            Body of the post 3.
            
            </div>
        
     </body>
</html>
